core-feature-pack/src/main/resources/content/bin/common.bat

```diff
@@ -2,6 +2,10 @@
 call %*
 goto :eof
 
+:setJava9_plus
+    "%JAVA%" --add-modules=java.se -version >nul 2>&1 && (set JAVA_9_PLUS=true) || (set JAVA_9_PLUS=false)
+goto :eof
+
 :setDefaultModularJvmOptions
   "%JAVA%" --add-modules=java.se -version >nul 2>&1 && (set MODULAR_JDK=true) || (set MODULAR_JDK=false)
```

Comment:
Code duplication with setJava9_plus

Btw, what is the reason for renaming MODULAR_JDK var to JAVA_9_PLUS var?

Reply:
I agree we probably should just use the `MODULE_JDK` variable. My guess however is that the unified GC logging was added in Java 9 which is why the variable name is `JAVA_9_PLUS`.

Reply:
@marekkopecky this is not renaming :) JAVA 9+ information is necessary for logging. While logging there can be added some JAVA_OPTS options which are then taken as input for modular JDK. So that's the reason why I had to split this function in two parts.

The duplicate code is of course wrong. It should be done in the same way as in common.sh. I will fix it.

Reply:
Line 10 uses MODULAR_JDK, line 6 JAVA_9_PLUS. Both means the same (both are created by the same command). We should have one "conventional" name for both. Currently, WF uses MODULAR_JDK ... I understand the idea for logging (unified GC logging was addid in JDK9, so you name it `JAVA_9_PLUS`, but anyway, we should be consistent and use just one variable name.